x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -91,5 +95,36 @@ export function StickyTransactionProperties({
     }
   ];
 
+  if (errorCount !== undefined) {
```

Comment:
the reason `errorCount` could be undefined is because it's not passed into the span flyout instance of the sticky properties. @formgeist do we want the related errors to be displayed in the span flyout for transactions as well? or just on the details page?

Getting it to show up in the span flyout is much more involved since, it is only passed transaction data that is in the waterfall. if transactions in the water fall were to show related errors, then we would need to do more related error lookups either on the server or client.

Reply:
Frankly I think this design solution was done before we finished up DT, which is why the related errors isn't really a part of the flyout. IMO it _should_ be there, since the flyout reflect the same information as the detail page, and it would be very useful when navigating a trace, opening a given transaction and seeing an error associated with it. 
Eventually we'll probably make this even more clear from the timeline view (show a highlight on the transaction or span), and even possibly have the error itself be shown in the flyout, so you don't have to go to the error group list first, but that's all to be looked later.

Is it worth investigating how to get it added to the flyout, or should we open a new ticket and make iteration on top of this in there? At least we'll have this current implementation done. Unless the work in the flyout will mean refactoring what's in the detail view.

FWIW I'm sorry this (again) is one of those times where the ticket should have groomed the solution a little more thoroughly and made adjustments based on the current implementation and worked from there. Unfortunately that's a common case when we're just picking something up from the backlog that's been sitting there for awhile.

Reply:
@ogupte Sorry, I must have missed this when reading your original comment, but you were talking about the span details flyout, and the related errors link isn't going in there. The related errors should be in the Transaction detail flyout, not sure if that's part of this implementation, but as I originally commented, this is something that wasn't entirely detailed in the design from the beginning.